pkg/web/httpservers.go

```diff
@@ -88,11 +89,11 @@ func NewHttpServer(
 	}
 
 	if enableProf {
-		profiler := &traceProfiler{}
 		routes = append(routes,
-			route{path: "/memprof", handler: profiler.MemProf, method: "POST", name: "profmem_post"},
-			route{path: "/pprof", handler: profiler.PProf, method: "POST", name: "profpprof_post"},
-			route{path: "/trace", handler: profiler.Trace, method: "POST", name: "proftrace_post"},
+			route{path: "/debug/pprof/", handler: pprof.Index, method: "GET", name: "pprof_index"},
```

Comment:
- [ ] removes trace

```
			route{path: "/debug/pprof/trace", handler: pprof.Trace, method: "GET", name: "pprof_trace"},
```

Reply:
ah, sorry. I thought it would be captured by the `pprof.Index` call.